src/browser/ui/React.js

```diff
@@ -116,7 +116,8 @@ if (__DEV__) {
 
     // If we're in IE8, check to see if we are in combatibility mode and provide
     // information on preventing compatibility mode
-    var ieCompatibilityMode = document.documentMode && document.documentMode < 8;
+    var ieCompatibilityMode = document.documentMode &&
+                              document.documentMode < 8;
```

Comment:
Put the whole RHS on a single new line, that's prefereable to this (there's something similar elsewhere but it's not in `src/` so I don't really care)